preview_generator/preview/builder/office__libreoffice.py

```diff
@@ -63,7 +63,7 @@ def _convert_to_pdf(
 
 def convert_office_document_to_pdf(
     file_content: typing.IO[bytes],
-    input_extension: str,  # example: '.dxf'
+    input_extension: typing.Optional[str],  # example: '.dxf'
```

Comment:
Do we allow None here ?

Reply:
It happen that this could have been None, yes:

If it was Falsy (None or the empty string), the following if would be executed:
```
if not input_extension:
    input_extension = mimetypes.guess_extension(mimetype)
```

but `guess_extension` can return None.

Also, the fact that there is an `if` checking if it's here proofes it's optional?